temporian/io/format.py

```diff
@@ -0,0 +1,49 @@
+"""Format of EventSet when importing/exporting them."""
```

Comment:
Is this "help" solution standard practice in other libs? I feel like it's a bit convoluted, both for the user (who needs to run a command to get the info they want - I'd rather link them to a page in our docs for example) and for us, who need to maintain this extra code and remember where it lives (to improve this we could name all help files "help.py" stored in corresponding directories). Might be worth it to discuss this in tomorrow's meeting!